implementations/rust/ockam/ockam_command/src/trust_context/show.rs

```diff
@@ -0,0 +1,39 @@
+use clap::Args;
+use ockam_api::cli_state::traits::StateDirTrait;
+
+use crate::{docs, CommandGlobalOpts};
+
+const LONG_ABOUT: &str = include_str!("./static/show/long_about.txt");
+const AFTER_LONG_HELP: &str = include_str!("./static/show/after_long_help.txt");
+
+/// Show the details of a trust context
+#[derive(Clone, Debug, Args)]
+#[command(
+    long_about = docs::about(LONG_ABOUT),
+    after_long_help = docs::after_help(AFTER_LONG_HELP)
+)]
+pub struct ShowCommand {
+    /// Name of the trust context
+    pub name: Option<String>,
+}
+
+impl ShowCommand {
+    pub fn run(self, opts: CommandGlobalOpts) {
+        if let Err(e) = run_impl(opts, self) {
+            eprintln!("{e}");
+            std::process::exit(e.code());
+        }
+    }
+}
+
+fn run_impl(opts: CommandGlobalOpts, cmd: ShowCommand) -> crate::Result<()> {
+    let name = cmd
+        .name
+        .unwrap_or(opts.state.trust_contexts.default()?.name().to_string());
+    let state = opts.state.trust_contexts.get(&name)?;
+    println!("Trust context:");
```

Comment:
Let's continue to use `opts.terminal.write_line` for these? 

(I have a seperate pr that update it to not write in cases of tty, and json output)

Reply:
Sure, I thought I did it, my bad!